src/entries/popup/hooks/useRestoreNavigation.ts

```diff
@@ -74,7 +74,11 @@ const RESTORE_NAV_MAP: Record<string, string[]> = {
   [ROUTES.SETTINGS__NETWORKS]: [ROUTES.SETTINGS, ROUTES.SETTINGS__NETWORKS],
   [ROUTES.SETTINGS__NETWORKS__CUSTOM_RPC]: [
     ROUTES.SETTINGS,
-    ROUTES.SETTINGS__NETWORKS__CUSTOM_RPC,
+    ROUTES.SETTINGS__NETWORKS,
```

Comment:
State restoration doesn't currently support dynamic routes, so this line was actually causing a little bug. This would take users back to the add network form instead of the add rpc form, because the chain id location state does not exist after restoration.

Might be worth going back and supporting this later, but for now I figured best to just drop the user back on the network settings page.

Reply:
Just for the record, this has been fixed in future commits